.github/workflows/reusable-module-prep-release.yml

```diff
@@ -81,53 +83,133 @@ jobs:
         run: composer validate
 
       - name: Install PHP Dependencies
-        run: composer install --no-dev --optimize-autoloader
+        uses: ramsey/composer-install@a2636af0004d1c0499ffca16ac0b4cc94df70565 # v3.1.0
+        with:
+          composer-options: "--no-dev --optimize-autoloader"
```

Comment:
I like to add `--verbose` etc. in workflow steps so when something does go wrong, there is more information to help debug